### ISSUE TITLE
The accuracy of CIFAR10-dvs dataset

### ISSUE THREAD

**Issue by junkangfang** [BICLab/Spike-Driven-Transformer#14]
Hello, I have a question about the accuracy of CIFAR10-dvs  datasets. I've tried  given configs , but the accuracy is lower than your proposed results. The accuracy of CIFAR10-dvs and gesture is around 74% and under 99% respectively.

So, I just wonder if there are other configs or settings like augmentation when you got the proposed accuracy,or 
please provide the 16-frame configuration file of cifar10-dvs.

Thank you.

**Comment by jkhu29** [comment]
#### For CIFAR10-DVS

As there is currently no official dataset splitting method for CIFAR10DVS, the training set and test set will be randomly split in each experiment. This may result in some variation in the experimental results across different hardware.

BTW, incorrect parameter settings most likely cause a low accuracy of 74%. Here we provide some key hyper-parameters for training on CTFAR10-DVS:
1. Open the `CutOut` augment;
2. Use `TET`;
3. Open `EMA`;
4. Close `fp16 (amp)`;
5. Longer training (larger epochs).

We also found that accurately determining the learning-rate in CIFAR10-DVS is challenging. You could use open-source hyperparameter search programs, such as [optuna](https://github.com/optuna/optuna), to find the best learning-rate on your hardware.

If you are still unable to achieve an accuracy higher than 79%, I suggest you use another [codebase](https://github.com/ZK-Zhou/spikformer/tree/main/cifar10dvs) for your experiments. In our subsequent experiments, we found that this codebase may offer a potential avenue for achieving better DVS recognition results.

#### For Gesture

You can also use the above hyperparameter tuning strategy to improve the Gesture results. However, given the limited size and noisy input signal (of these DVS datasets), overfitting models on these small datasets may not be essential.

**Comment by junkangfang** [comment]
thanks！！
